src/COLLOID/pair_lubricate.cpp

```diff
@@ -142,7 +142,7 @@ void PairLubricate::compute(int eflag, int vflag)
     Ef[2][2] = h_rate[2]/domain->zprd;
     Ef[0][1] = Ef[1][0] = 0.5 * h_rate[5]/domain->yprd;
     Ef[0][2] = Ef[2][0] = 0.5 * h_rate[4]/domain->zprd;
-    Ef[1][2] = Ef[2][1] = 0.5 * h_rate[3]/domain->zprd;
+    Ef[1][2] = Ef[2][1] = 0.5 * h_rate[3]/domain->xprd;
```

Comment:
Not clear on why this change is correct.

Reply:
@sjplimp you are correct. I was overeager and too careless to apply this change. It clearly contradicts the comments above this code section. I have reverted this change from the branch and will wait how the issue will be resolved before adding it again.